src/chainer.js

```diff
@@ -12,7 +15,7 @@ function chainer(functions) {
     }
 
     return result;
-  }
-}
```

Comment:
The semicolon is unnecessary here. In JavaScript, it's optional to use a semicolon at the end of a statement, but it's a good practice to use them to avoid any potential errors.